### DISCUSSION TITLE
Move all customizations to single env file

### DISCUSSION THREAD

**Discussion by osnipezzini** [rustdesk/rustdesk#2078] [General]
A good practice for companies is to use their own system. With that in mind I would like to suggest an improvement, to centralize all possible customization fields in a single file.

When we are going to compile we need to change many files, and many of them can present problems if altered improperly.

In a single file, we could customize the application ID, name, version, logo, public key, servers, among others.

**Comment by luizjr** [comment]
This idea is brilliant.
As it is, it's pretty tight.

If you put them as environment variables, it makes it much easier for companies to use them more and more.

**Comment by pedjas** [comment]
That would be practical. When using clients for standalone servers ti is good to have option to make visual difference from public server, like changing icon or overall color, but also host address.

**Comment by osnipezzini** [comment]
Any news about this? ADM just ignored this ?

Instead this, they replace variables by strings: https://github.com/rustdesk/rustdesk/commit/8a68974f4f1fa17073a82c331075ed5dd2ca4a0a
